src/Backend/Modules/Tags/Engine/Model.php

```diff
@@ -73,10 +63,18 @@ public static function exists($id)
      */
     public static function existsTag($tag)
     {
-        return (BackendModel::getContainer()->get('database')->getVar(
-                    'SELECT i.tag FROM tags AS i  WHERE i.tag = ?',
-                    array((string) $tag)
-                ) != '');
+        /** @var Tag Retrieve the tag */
+        $item = BackendModel::get('doctrine.orm.entity_manager')
```

Comment:
Not exactly the same. Here we search for a "tag" string. In get($id) we use an integer.